cmd/src/cli/run.rs

```diff
@@ -0,0 +1,67 @@
+use cli::{self, package};
+use error::DefaultResult;
+use holochain_container_api::{
+    config::*,
+    container::Container,
+};
+
+/// Starts a small container with the current application running
+pub fn run(package: bool) -> DefaultResult<()> {
+    if package {
+        cli::package(true, Some(package::DEFAULT_BUNDLE_FILE_NAME.into()))?;
+    }
+
+    let agent_config = AgentConfiguration {
+        id: "hc-run-agent".into(),
+        key_file: "hc_run.key".into(),
+    };
+
+    let dna_config = DNAConfiguration {
+        id: "hc-run-dna".into(),
+        file: package::DEFAULT_BUNDLE_FILE_NAME.into(),
+        hash: "Qm328wyq38924ybogus".into(),
+    };
+
+    let instance_config = InstanceConfiguration {
+        id: "hc-run-instance".into(),
+        dna: "hc-run-dna".into(),
+        agent: "hc-run-agent".into(),
+        logger: Default::default(),
+        storage: StorageConfiguration::Memory,
+    };
+
+    let interface_config = InterfaceConfiguration {
+        id: "websocket-interface".into(),
+        driver: InterfaceDriver::Websocket { port: 8888 },
+        admin: true,
+        instances: vec![InstanceReferenceConfiguration {
+            id: "hc-run-instance".into()
+        }],
+    };
+
+    let base_config = Configuration {
+        agents: vec![agent_config],
+        dnas: vec![dna_config],
+        instances: vec![instance_config],
+        interfaces: vec![interface_config],
+        ..Default::default()
+    };
+
+    let mut container = Container::with_config(base_config);
+
+    container.start_all_interfaces();
+    container.start_all_instances()?;
+
```

Comment:
@sphinxc0re we need just a tiny bit of CLI feedback here...
tell the user the service is running

Reply:
And also tell the user that they can type `exit` to exit